meetings/2017-12-13.md

```diff
@@ -0,0 +1,94 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-12-13
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=1sHufGWR_WU
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/435
+* **Minutes Google Doc**: https://docs.google.com/document/d/1oWrjw845TDCAYCZHkYv51oyA_d-jxzcGe0hvwtf1_Io
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Michaël Zasso @targos (TSC)
+* Trevor Norris @trevnorris (TSC)
+* Rich Trott @Trott (TSC)
+
+
+
+## Agenda
+
+### Announcements
+* Collab summit Berlin, most likely on May 31 - Friday, June 1, 2018. The 2 days prior to JSConf.eu [#56](https://github.com/nodejs/summit/issues/56).
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* events: add off alias to removeListener [#17156](https://github.com/nodejs/node/pull/17156)
+
+Rich: GitHub conversation did not get “unstuck”. Move to vote.
+
+
+* DO NOT MERGE: \[module\] Revert checking for package.json before extensions [#15015](https://github.com/nodejs/node/pull/15015)
+
+Michaël: Need an update from Bradley
+
+Franzi: Leave it for next week’s meeting.
+
+Rich: Invite Bradley as a guess.
+
+* build: update minimum kernel version to 3.10.0 [#14795](https://github.com/nodejs/node/pull/14795)
+
+Rich: Given this and the last issue, maybe we should invite Ruben to the meeting as an observer too next time to frame these issues.
+
+Franzi: Agreed. We need more people to decide and frame these. Maybe we should invite Gibson too.
+
+### nodejs/TSC
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+   * N-API - continue to work on moving out of ‘experimental’.  Looking to use bcrypt and
+     one of Anna’s modules for production use.  Have made some progress on investigating
+     how node-pre-gyp will work, focus on tests and doc continues.
+
+  * Moderation: Meeting today. Will have a “real” update next time.
+
+  * V8: Opened PR for 6.4. Not expecting major issues with this update.
+
+  * Module: There was the blog post on the 
```

Comment:
Add:

 Node.js Foundation's Medium Publication [The Current State of Implementation and Planning for ESModules](https://medium.com/the-node-js-collection/the-current-state-of-implementation-and-planning-for-esmodules-a4ecb2aac07a)